### DISCUSSION TITLE
EasyBrightness for ios 14

### DISCUSSION THREAD

**Discussion by K4rolek** [community/community#27740] [Other Feature Feedback, Questions, & Ideas]
hello can you update tweak to ios 14? it doesn't work, greetings

**Comment by MrSarno** [comment]
Hi @K4rolek,

This is a public forum about GitHub itself. It is incredibly unlikely anyone capable of helping you with this will find your post here.

If you follow [this link](https://github.com/gilesgc/EasyBrightness/issues/new) you can open an issue for the project which I believe to be the one you are posting about. That way the developer of EasyBrightness will receive a notification about your query, and will hopefully be able to help you accordingly.

A lot of us keep an eye on new topics here to try to help people. I'd really appreciate it if you could mark this response as the answer to your question, as that way, it won't appear in [the list](https://github.com/orgs/community/discussions?discussions_q=is%3Aunanswered) of unanswered questions needing to be resolved.

Thanks!
